portal.go

```diff
@@ -1966,3 +1967,42 @@ func (portal *Portal) UpdateInfo(source *User, meta *discordgo.Channel) *discord
 	}
 	return meta
 }
+
+func (portal *Portal) ForwardBackfill(source *User) error {
+	portal.log.Debugln("Checking for missing messages to fill")
+	lastMessage := portal.bridge.DB.Message.GetLast(portal.Key)
+	if lastMessage == nil {
+		portal.log.Debugln("No last message in portal, can't forward backfill")
+		return nil
+	}
```

Comment:
This needs to check the last message ID from the channel struct before backfilling

The discord docs say that the last message ID "may not point to an existing or valid message or thread", so it should probably do an integer comparison (because discord message IDs are ordered snowflakes) and only try to backfill when the channel struct value is higher than the database value.